### DISCUSSION TITLE
Flip patient name

### DISCUSSION THREAD

**Discussion by johnson-cloud** [dcm4che/dcm4chee-arc-light#4323] [General]
Dear all,

I am passing correct patient names in hl7 to mwl.

But on modality patient family name is coming first then first name

I want to switch to patient name first, middle then last name on modality mwl display

**Comment by vrindanayak** [comment]
In archive, the patient name with family name first followed by rest of the components follow the standard conventions as specified in 
- [DICOM PS3.5](https://dicom.nema.org/medical/dicom/current/output/html/part05.html#sect_6.2.1.1)
- [HL7 - PID.5](https://hl7-definition.caristix.com/v2/HL7v2.5.1/Fields/PID.5)

> I want to switch to patient name first, middle then last name on modality mwl display

You would need to check with modality technician.
